### PR TITLE
testing a pull request

### PR DESCRIPTION
**Pull request by o-weir**

### CODE REVIEW

File: oli.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hi christian
```
Comment (christianolivodiaz) [inline oli.txt:1]: - Thank you for the request

Comment (christianolivodiaz) [inline oli.txt:1]:
```suggestion
i ate too much
```

**Comment by o-weir** [comment]
testing some stuff

**Review by christianolivodiaz — COMMENTED** [review]
Good luck w BCS exam

**Comment by o-weir** [comment]
cheers good luck to you too

**Pull request merged**